packages/ozone/tests/__snapshots__/get-account-timeline.test.ts.snap

```diff
@@ -0,0 +1,36 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`account timeline Returns entire timeline of events for a given account 1`] = `
+Array [
+  Object {
+    "count": 1,
+    "eventSubjectType": "record",
+    "eventType": "tools.ozone.moderation.defs#modEventReport",
+  },
+  Object {
+    "count": 1,
+    "eventSubjectType": "record",
+    "eventType": "tools.ozone.moderation.defs#modEventTag",
+  },
+  Object {
+    "count": 2,
+    "eventSubjectType": "account",
+    "eventType": "tools.ozone.moderation.defs#modEventReport",
+  },
+  Object {
+    "count": 1,
+    "eventSubjectType": "account",
+    "eventType": "tools.ozone.moderation.defs#modEventTag",
+  },
+  Object {
+    "count": 1,
+    "eventSubjectType": "account",
+    "eventType": "tools.ozone.moderation.defs#modEventTakedown",
+  },
+  Object {
+    "count": 1,
+    "eventSubjectType": "account",
+    "eventType": "plc_operation",
+  },
```

Comment:
Not sure if it's an issue with the snapshots or if there's a small bug, but still seeing `plc_operation` used as an `eventType`.

Reply:
yep, needed to update the snapshot.